packages/valory/skills/decision_maker_abci/utils/scaling.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2024 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+
+"""This package contains helpers for scaling operations."""
+
+
+from typing import List, Tuple
+
+
+def min_max(li: List[float]) -> Tuple[float, float]:
+    """Get the min and max of a list."""
+    if not li:
+        raise ValueError("The list is empty.")
+
+    min_ = max_ = li[0]
+
+    for num in li[1:]:
+        if num < min_:
+            min_ = num
+        elif num > max_:
+            max_ = num
+
+    return min_, max_
+
+
+def scale_value(
+    value: float,
+    min_max_bounds: tuple[float, float],
+    scale_bounds: tuple[float, float] = (0, 1),
```

Comment:
We cannot change the target range to a different value from [0, 1] because it will impact now the value of the dynamic kelly fraction, so I request removing this possibility from here and simplifying this function

Reply:
There is no reason to remove that from the util method. The change is blocked upstream.